index.js

```diff
@@ -8,43 +8,57 @@ function testParameter(name, filters) {
 
 module.exports = (urlString, opts) => {
 	opts = Object.assign({
+		defaultProtocol: 'http:',
 		normalizeProtocol: true,
-		normalizeHttps: false,
-		normalizeHttp: false,
-		stripFragment: true,
+		forceHttp: false,
+		forceHttps: false,
+		stripHash: true,
 		stripWWW: true,
 		removeQueryParameters: [/^utm_\w+/i],
 		removeTrailingSlash: true,
 		removeDirectoryIndex: false,
 		sortQueryParameters: true
 	}, opts);
 
+	// Backwards compatibility
+	if (opts.normalizeHttps) {
+		opts.forceHttp = true;
+	}
+
+	if (opts.normalizeHttp) {
+		opts.forceHttps = true;
+	}
+
+	if (opts.skipFragment) {
+		opts.skipHash = true;
+	}
```

Comment:
It's `strip`, not `skip`.

Can you add back at least one test for each of these so we can ensure backwards compatibility is preserved?

Reply:
done